### ISSUE TITLE
Blacklight.do_bookmark_toggle_behavior(); is creating multiple label/checkboxes

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/SearchWorks#454]
On the gallery view Blacklight.do_bookmark_toggle_behavior(); is being called several times creating multiple form elements.
